designer/client/src/reducers/graph/utils.ts

```diff
@@ -77,6 +79,33 @@ export function prepareNewNodesWithLayout(
     };
 }
 
+export function removeStickyNoteFromLayout(state: GraphState, stickyNoteId: number): { layout: NodePosition[]; stickyNotes: StickyNote[] } {
+    const { layout } = state;
+    const stickyNoteLayoutId = createStickyNoteId(stickyNoteId);
+    const updatedStickyNotes = state.stickyNotes.filter((n) => n.noteId !== stickyNoteId);
+    const updatedLayout = updatedStickyNotes.map((stickyNote) => {
+        return { id: createStickyNoteId(stickyNote.noteId), position: stickyNote.layoutData };
```

Comment:
Why are you recreating this stickyNoteId? `noteId` hasn't changed, so why are you regenerating `id` with `createStickyNoteId`?

Reply:
You are right, that was redundant - removed :+1: